src/models/event.js

```diff
@@ -1143,6 +1144,42 @@ utils.extend(MatrixEvent.prototype, {
     getTxnId() {
         return this._txnId;
     },
+
+    /**
+     * Get a copy/snapshot of this event. The returned copy will be loosely linked
+     * back to this instance, though will have "frozen" event information. Other
+     * properties of this MatrixEvent instance will be copied verbatim, which can
+     * mean they are in reference to this instance despite being on the copy too.
+     * Consumers should be wary of using fields which may mutate over time.
```

Comment:
Hmm, the wording is still a bit strange to me, maybe include an example of the sender profile...?